playbooks/task-get-resourceids.yml

```diff
@@ -24,7 +24,14 @@
     listener_id: "{{resource_ids.results[2].stdout}}"
     healthmonitor_id: "{{resource_ids.results[3].stdout}}"
     member_ids: "{{resource_ids.results[4].stdout_lines}}"
-    project_id: "{{resource_ids.results[5].stdout}}"
+
+- set_fact:
+    project_id: "{{  resource_ids.results[5].stdout  }}"
+  when: tenant_id is not defined
```

Comment:
tenant_id is just project_id. Could we uses project_id.
furthermore, don't change main logic just because only one test need it.